tests/main/special-home-can-run-classic-snaps/task.yaml

```
@@ -36,7 +36,10 @@ restore: |
             # TODO: remove the apt key added above, but how?
             ;;
         postgres)
-            apt autoremove --purge -y postgresql
+            # the first one will fail because of ... a race with /etc/alternatives?
```

Nitpick: capitalise sentences please
```suggestion
            # The first one will fail because of ... a race with /etc/alternatives?
```